app/style/common/mixins.less

```diff
@@ -242,10 +242,6 @@
   visibility: hidden;
 }
 
-.visible {
-  visibility: visible;
-}
-
```

Comment:
Why removing a mixin?

Reply:
It is not used anywhere and actually clashes with simplebar.